practices/git_flow.md

```diff
@@ -0,0 +1,77 @@
+---
+Slug: git-flow-workflow
+Aspect: "processes"
+stages:
+- development
+Short_description: Git Flow Workflow is a branching system for Git. It helps the team to better control and add different project versions.
```

Comment:
I'd change `...is a branching system for Git.` to `...is opinionated a branching system for Git.` or `...is specific a branching system for Git.` to say that it's not the common workflow.